coa/pkg/apis/v1alpha2/providers/reporter/k8s/k8sreporter_test.go

```diff
@@ -38,3 +47,96 @@ func TestGet(t *testing.T) {
 	}, false)
 	assert.Nil(t, err)
 }
+
+func TestInitWithMap(t *testing.T) {
```

Comment:
Update the test name to a more appropriate name.

Reply:
Fixed